tests/modules/virtualisation/podman/docker-compat.nix

```
@@ -0,0 +1,15 @@
+{ config, pkgs, lib, ... }:
+
+{
+  config = {
+    services.podman = {
+      enable = true;
+      dockerCompat = true;
+      dockerSocket = true;
+    };
+    nmt.script = ''
+      assertFileIsExecutable home-path/bin/docker
+      assertFileContains home-path/etc/profile.d/hm-session-vars.sh "DOCKER_HOST"
+    '';
+  };
```

```suggestion
  config = {
    services.podman = {
      enable = true;
      dockerCompat = true;
      dockerSocket = true;
    };
    
    test.stubs.podman = { };
    
    nmt.script = ''
      assertFileIsExecutable home-path/bin/docker
      assertFileContains home-path/etc/profile.d/hm-session-vars.sh "DOCKER_HOST"
    '';
  };
```

Can you add stub packages for the other packages used in the configuration?